css/main.css

```diff
@@ -100,15 +100,15 @@ div#nav li a{
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divHola h1{
+div#div-hola h1 {
     color: #00ADBB;
     font-size: 80px;
 }
 
-div#divHola{
+div#div-hola {
     text-align: center;
     display: block;
-    background-color:#ffffff;
+    background-color: #ffffff;
```

Comment:
Can we add the value of the hex colors in vars? It would help reused the values in all your classes.

Reply:
done.